src/ProxyClient/Http/InvalidationRequest.php

```diff
@@ -21,7 +21,7 @@
 class InvalidationRequest implements RequestInterface
 {
     use RequestDecorator;
```

Comment:
looks like we are missing a require statement. this would be in https://github.com/php-http/message-decorator - i guess we still want to use that, right? maybe @sagikazarmark split that up after @ddeboer built the initial version of this?